ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js

```diff
@@ -17,12 +17,14 @@ describe("Form with Submit Button", () => {
 
     const pagePath = "content/forms/af/core-components-it/samples/actions/submit/basic.html"
     const customSubmitPagePath = "content/forms/af/core-components-it/samples/actions/submit/customsubmit/basic.html"
+    const customSubmitLocalisationPagePath = "content/forms/af/core-components-it/samples/actions/submit/customsubmit/thankyoumessage_localisation.de.html"
```

Comment:
Can you use the existing submit collateral for these test (add relevant dictionaries to it) ? I don't think you need an another collateral for this use-case